draft-irtf-cfrg-vdaf.md

```diff
@@ -3848,12 +3920,13 @@ by a wrapper gadget, `QueryGadget`, listed in {{gadget-wrappers}}. This gadget
 also evaluates the gadget polynomial for each gadget invocation in order to
 produce the gadget's output. Then compute the wire polynomials from the
 recorded values.
+Note that these polynomials are in the value (Lagrange) representation
+by construction.
```

Comment:
Here's a suggested rephrasing, to reflect that only the wire polynomials are constructed during the query process, while the gadget polynomials are received from the client.
```suggestion
Note that the wire polynomials are in the value (Lagrange) representation
by construction, and the gadget polynomials parsed from the proof share
are similarly transmitted in value (Lagrange) representation.
```